tests/unit/src/test_conditionsStore.cpp

```diff
@@ -235,18 +213,16 @@ SCENARIO( "Setting and erasing conditions", "[ConditionStore][ConditionSetting]"
 				REQUIRE( store.Get("myFirstVar") == 10 );
 				REQUIRE( store["myFirstVar"] == 10 );
 			}
-			THEN( "erasing the condition will make it disappear again" )
+			THEN( "erasing the condition is done by setting it to zero" )
```

Comment:
Setting it to zero and erasing aren't the same thing at runtime.
The entry still exists if it is set to zero, it just returns a value of zero, the same as if it were erased.
In terms of containers, "erase" generally means for the entry to no longer be present.

Assuming we already have tests for non-existent or zero values returning false, I'm not sure this test is necessary anymore. It's checking that, upon erasure, the relevant entry is removed from the `storage` container. Since that functionality no longer exists, and setting a condition to zero should not result in that entry being immediately removed from the `storage` container, there's not much point in this test anymore.

Reply:
Agreed, I changed the tests to explicit set tests.